lib/Common/Memory/RecyclerWriteBarrierManager.cpp

```diff
@@ -325,6 +335,11 @@ RecyclerWriteBarrierManager::WriteBarrier(void * address, size_t bytes)
     Assert(startIndex <= endIndex);
     memset(cardTable + startIndex, WRITE_BARRIER_PAGE_BIT | DIRTYBIT, endIndex - startIndex);
     GlobalSwbVerboseTrace(_u("Writing to 0x%p (CIndex: %u-%u)\n"), address, startIndex, endIndex);
+
+#if DBG
+    Recycler::WBSetBits((char*)address, (uint)bytes/sizeof(void*));
```

Comment:
>(uint) [](start = 40, length = 6)

Is this "(uint)" cast needed? Does it divide by "sizeof" type and again result in "size_t"?